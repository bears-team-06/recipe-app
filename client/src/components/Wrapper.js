import styled from 'styled-components'

const Wrapper = styled.div`
  display: flex;
  flex-direction: row;
  align-items: stretch;
`
export default Wrapper;