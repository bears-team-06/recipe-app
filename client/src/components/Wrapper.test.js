import React from "react";
import renderer from "react-test-renderer";
import Wrapper from "./Wrapper"

describe("Wrapper", () => {
    it("matches snapshot", () => {
        const WrapperVar = renderer
            .create(<Wrapper/>)
            .toJSON();
        expect(WrapperVar).toMatchSnapshot();
    });
})