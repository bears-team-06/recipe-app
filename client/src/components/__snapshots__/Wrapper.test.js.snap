// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wrapper matches snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-align-items: stretch;
  -webkit-box-align: stretch;
  -ms-flex-align: stretch;
  align-items: stretch;
}

<div
  className="c0"
/>
`;
